test/verify/check-system-info

```diff
@@ -828,6 +828,36 @@ password=foobar
         wait(lambda: progressValue(2) <= hog_usage - 15)
         self.assertGreater(progressValue(2), 10)
 
+    def testShutdownStatus(self):
+        m = self.machine
+        b = self.browser
+
+        self.login_and_go("/system")
+        b.wait_not_present("#system-health-shutdown-status")
+
+        # Schedule a reboot
+        m.execute("shutdown --reboot +10")
+        self.addCleanup(m.execute, "shutdown -c")
+
+        b.wait_in_text('#system-health-shutdown-status-text', "Scheduled reboot")
+
+        # Check that reloading still shows the reboot text
+        b.reload()
+        b.enter_page("/system")
+        b.wait_in_text('#system-health-shutdown-status-text', "Scheduled reboot")
+
+        # Cancel
+        b.click("#system-health-shutdown-status-cancel-btn")
+        b.wait_not_present('#system-health-shutdown-status')
```

Comment:
Let's also validate with a busctl call that this really made it into systemd, and `ScheduledShutdown` property is back to 0.

Reply:
Sure!